common/internal_model/src/main/scala/weco/catalogue/internal_model/locations/AccessStatus.scala

```diff
@@ -24,10 +24,10 @@ sealed trait AccessStatus extends EnumEntry { this: AccessStatus =>
   }
 }
 
-object AccessStatus extends Enum[License] {
+object AccessStatus extends Enum[AccessStatus] {
```

Comment:
Was this meant to slip in here?

Reply:
Oops. It was not. I can pull it into a separate PR if you think it's important, but it's just a typo I spotted while working on this and I forgot to pull into a separate PR.

Reply:
It looks like more than a typo? If it's a noop, all good to stay here.

Reply:
typo or copy/paste error. Should be a no-op, but CI will confirm.